src/app/client/src/app/app.component.ts

```diff
@@ -173,9 +173,12 @@ export class AppComponent implements OnInit, OnDestroy {
     while (child.firstChild) {
       child = child.firstChild;
     }
-    const pageTitle = _.get(child, 'snapshot.data.pageTitle') || _.get(child, 'snapshot.data.telemetry.pageid') || _.get(this.userService, 'rootOrgName');
+    let pageTitle = _.get(child, 'snapshot.data.pageTitle') || _.get(child, 'snapshot.data.telemetry.pageid') || _.get(this.userService, 'rootOrgName');
     if (pageTitle) {
+      pageTitle = this.instance + ": "+ pageTitle;
```

Comment:
@devendra-tarento 
this will impact core sunbird functionality.
it should be configurable 
CC: @vinukumar-vs @rajeevsathish @Krishnaj20

Reply:
@rajeshkumaravel It is just an enhancement irrespective of sunbird instance or adopter. In my opinion it should be applicable to all hence i did not make it configurable.

Reply:
@vinukumar-vs Can we consider @devendra-tarento 's opinion ?
CC: @vinukumar-vs @rajeevsathish @Krishnaj20

Reply:
@vinukumar-vs @Krishnaj20 @rajeevsathish Could you please provide your input on this. So that we can move ahead on this PR.